include/sys/arc_impl.h

```diff
@@ -744,6 +749,18 @@ typedef struct arc_stats {
 	kstat_named_t arcstat_mfu_ghost_evictable_metadata;
 	kstat_named_t arcstat_l2_hits;
 	kstat_named_t arcstat_l2_misses;
+	/*
+	 * Aligned size (in bytes) of L2ARC cached buffers by ARC state.
```

Comment:
These are allocated sizes. I'm not sure what aligned size means.

Reply:
Of course. I was carried away by the P2ALIGN function used to calculate the allocated size.